src/NodaTime.Test/Text/AnnualDateTypeConverterTest.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright 2018 The Noda Time Authors. All rights reserved.
+// Use of this source code is governed by the Apache License 2.0,
+// as found in the LICENSE.txt file.
+
+using JetBrains.Annotations;
+using NodaTime.Text;
+
+namespace NodaTime.Test.Text
+{
+    [UsedImplicitly]
+    public class AnnualDateTypeConverterTest : TypeConverterBaseTestBase<AnnualDate>
```

Comment:
The test suite appears to use a style of type-under-test-per-file, so I've tried to follow suit with these. That said, I could move them all into a single file (e.g. `TypeConverterTest.cs`) with each derived type as a separate method.